ci-operator/step-registry/etcd-encryption/vault-install/etcd-encryption-vault-install-commands.sh

```
@@ -70,18 +71,19 @@ helm upgrade --install vault hashicorp/vault \
   --timeout 10m
 
 #helm wait passes even vault pod is 0/1 Running. So, added the below wait to correctly verify the vault pod status
-echo "Waiting for Vault pod to be ready..."  
-oc wait --for=condition=ready pod/vault-0 -n "${VAULT_NAMESPACE}" --timeout=5m
+echo "Waiting for Vault pod to be ready..."
+oc wait --for=condition=ready pod/${VAULT_RELEASE_NAME}-0 -n "${VAULT_NAMESPACE}" --timeout=5m
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)